codice/ddf
# [2.13.x] DDF-4287 Registry node

#### What does this PR do?
It fixes the issue where a 403 error was displayed when attempting to access `Admin Consoles Catalog-> Node Information -> View links`. A new value was added to the to the pertinent blueprint, which fixes the issue.

#### Who is reviewing it? 
@kcover @emmberk @clockard @peterhuffer @mdang8 

#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@vinamartin

#### How should this be tested?
Verify that visiting `Admin Consoles Catalog-> Node Information -> View links` doesn't result in a 403 error.


#### What are the relevant tickets?
[DDF-4287](https://codice.atlassian.net/browse/DDF-4287)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
